Mathlib/Init/Algebra/Classes.lean

```diff
@@ -180,7 +180,7 @@ theorem lt_of_lt_of_incomp {α : Sort u} {lt : α → α → Prop} [IsStrictWeak
     [DecidableRel lt] : ∀ {a b c}, lt a b → ¬lt b c ∧ ¬lt c b → lt a c :=
   @fun a b c hab ⟨nbc, ncb⟩ =>
   have nca : ¬lt c a := fun hca => absurd (trans_of lt hca hab) ncb
-  Decidable.by_contradiction fun nac : ¬lt a c =>
+  Decidable.byContradiction fun nac : ¬lt a c =>
```

Comment:
Does this have anything to do with the PR?

Reply:
It is the fallout of the shake linter.